qutebrowser/mainwindow/prompt.py

```diff
@@ -642,8 +643,16 @@ def _init_fileview(self):
         self._file_model.directoryLoaded.connect(
             lambda: self._file_model.sort(0))
 
+    def _show_error(self, msg):
+        log.prompt.error(msg)
+        QToolTip.showText(self._lineedit.mapToGlobal(QPoint(0, 0)), msg)
+
```

Comment:
I'd prefer "normal" error messages too, I just wasn't sure if that change would be desirable or  hard to implement. But I'll switch the order!